### DISCUSSION TITLE
Control accordion header click area

### DISCUSSION THREAD

**Discussion by yachtsman-dev** [primefaces/community#2148] [PrimeReact]
I am trying to customise Primereact Accordion header with custom html. [Acordion](https://primereact.org/accordion/) 
Want to achieve something like in attached image. When i add `InputSwitch` right side it's not updating switch status (on /off) but instead its closing and opening Accordion tabs. Is there any way i can bound/control accordion tab click area so i can click on switch button?
![Screenshot 2024-06-10 at 10 05 39](https://github.com/primefaces/community/assets/101167142/7bcd28fe-0bed-426c-84ae-d5dff3f6f83f)

**Comment by melloware** [comment]
Can you put together a StackBlitz reproducer showing the issue?
